decoders/connector/lansitec/precision-platinum-sensor/v1.0.0/payload.js

```diff
@@ -0,0 +1,168 @@
+// PT100
```

Comment:
New decoders that are added must be made in TypeScript and must have a unit test.